hydroflow_lang/src/graph/flat_graph_builder.rs

```diff
@@ -29,6 +29,29 @@ enum GraphDet {
     Undetermined(Ident),
 }
 
+/// Variable name info for each ident, see [`FlatGraphBuilder::varname_ends`].
+#[derive(Debug)]
+struct VarnameInfo {
+    /// What the variable name resolves to.
+    pub ends: Ends,
+    /// Set to true if the varname reference creates an illegal self-referential cycle.
+    pub illegal_cycle: bool,
+    /// Set to true once the in port is used. Used to track unused ports.
+    pub inn_used: bool,
```

Comment:
Is `inn` a typo?

Reply:
Mhm, when I originally made this file I had `inn` to match the length of `out` in the code formatting, maybe silly

Reply:
It's fine. Just difficult to keep it uniform throughout if there are more people working on the code base. I don't feel strongly though.

Reply:
Yeah it's not even consistent with just me working on it

Reply:
If you wanna change it to be the same everywhere, it would best go in a separate CR.